### ISSUE TITLE
Hide when overlapping 

### ISSUE THREAD

**Issue by kvasnyj** [josephernest/bigpicture.js#19]
Hello, 
Is it possible to hide foreground text when zoom in to not overlapping background text?

For example hide "Newton" label when zoom in:
![image](https://user-images.githubusercontent.com/229747/50818765-59ef8600-1328-11e9-8af2-c92185e3563c.png)

**Comment by josephernest** [comment]
It it a good idea... but how to define the concept of overlapping / foreground / background?
Currently textboxes only have `x`, `y` position and `size`.

What would be the rule: in which case should a big text be hidden, and when should it reappear?

**Comment by kvasnyj** [comment]
What if when `size` greater than `threshold`?

**Comment by josephernest** [comment]
Yes but some people use big letters as an artistic effect, like a very big `O` (like a big circle) and some text inside, etc. :)
See https://www.youtube.com/watch?v=52ilOpxWYuc

**Comment by kvasnyj** [comment]
Sure, in this case, `threshold=0` by default
